bin/container.sh

```diff
@@ -175,39 +166,24 @@ start_cassandra() {
 }
 
 console_cassandra() {
-  wcip=$(docker inspect --format "{{ .NetworkSettings.Networks.${docker_network}.IPAddress }}" ${project}-cassandra)
-
-  docker run --net=${docker_network} -it --rm ${cassandra} cqlsh ${wcip} || \
-    usage "unable to run command: docker run --net=${docker_network} -it --rm ${cassandra} cqlsh ${wcip}" 1
+  docker exec -it ${project}-cassandra cqlsh || \
```

Comment:
my thoughts around using 'run' vs 'exec' was that it ensured a "new" cqlsh client could network connect to the cas container, vs just starting a cqlsh client on the existing cas container ... which doesn't seem like a real network-reachability test. make sense?

thoughts?